.github/workflows/docsite-publish-chromatic.yml

```diff
@@ -44,7 +44,10 @@ jobs:
         run: yarn install --frozen-lockfile
 
       - name: Build dependencies
-        run: yarn build --to @fluentui/public-docsite-v9
+        uses: ./.github/actions/build-to
+        with:
+          workspaces: '@fluentui/public-docsite-v9'
```

Comment:
**suggestion:** can this be an array (based on the naming), if not can we rename this to `workspace` ?

Reply:
how would that look like ? `@fluentui/public-docsite-v9,@fluentui/foo,@fluentui/bar` ?

Reply:
Space separated, `@fluentui/public-docsite-v9 @fluentui/foo,@fluentui/bar`. There's the option of using JS arrays and then JSON.parse them, but I think that will make them worse: https://github.com/community/community/discussions/11692#discussioncomment-3541856

Reply:
yeah lets avoid that. last thing - can you please add documentation to that input so one can find how to define multiple workspaces ? ty

Reply:
Added a comment to give more details about why we chose this approach and improved the field's description.